### DISCUSSION TITLE
Connecting to managed Redis over TLS

### DISCUSSION THREAD

**Discussion by ymitsos** [getredash/redash#6564] [Q&A] [answered]
Hi, 

We are trying to deploy Redash using the provided helm chart inside an AWS environment. We want to take advantage of a serverless Redis that is available under the ElasticCache AWS service. We also have a hard requirement to use TLS in transit. 
Said that, it seems that there is no such value inside the helm chart to force TLS communication. Are we missing something? Is this use case not supported?

**Comment by guidopetri** [comment, marked as answer]
I don't think anyone has touched the Redis-related code in a while. Someone else created #6565 (possibly they're related to you somehow?) that covers the same use case.

We'd gladly welcome a PR for adding this, if you'd like to implement it!

Closing this so that we can keep the discussion in #6565.
